dotcom-rendering/src/web/components/DynamicSlowMPU.stories.tsx

```diff
@@ -22,7 +23,7 @@ export default {
 	},
 };
 
-const bigs = trails.slice(0, 3);
+const bigs = trails.slice(0, 3) as [TrailType, TrailType, TrailType];
```

Comment:
This is a story, if the type assertion is wrong, it will differ at runtime when building storybook.